tests/testing_utils.py

```diff
@@ -78,7 +78,7 @@ def require_torch_multi_gpu(test_case):
         return test_case
 
 
-def require_multi_accelerator(test_case):
+def require_torch_multi_accelerator(test_case):
```

Comment:
this is actually for torch multi-accelerator, since it uses `torch_device`, change the name to reflect the fact and align the name convention w/ `require_torch_multi_gpu.`